Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>DB_HOST = localhost\r\nDB_DATABASE = flight_game\r\nDB_USER = root\r\nDB_PASSWORD = password\r\n\r\nFLIGHT_RANGE = 2778  # km
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
--- a/.env	(revision ee80230d92cd37d0dabca912b083f2c6199016e3)
+++ b/.env	(date 1696337033019)
@@ -1,6 +1,6 @@
 DB_HOST = localhost
 DB_DATABASE = flight_game
 DB_USER = root
-DB_PASSWORD = password
+DB_PASSWORD = 3042
 
 FLIGHT_RANGE = 2778  # km
\ No newline at end of file
Index: wege.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/wege.py b/wege.py
new file mode 100644
--- /dev/null	(date 1696337033049)
+++ b/wege.py	(date 1696337033049)
@@ -0,0 +1,23 @@
+import random
+
+import helpers.connector as connector
+
+my_cursor = connector.mydb.cursor(dictionary=True, buffered=True)
+
+weather_list = ['sadetta', 'pilvistä', 'aurinkoista', 'lumisadetta', 'puolipilvistä']
+
+
+def weather_adder(list):
+    amount = 10
+    while amount > 0:
+        weather = random.choice(list)
+        temp = random.randrange(-15, 31, 5)
+        weatherex = 'SELECT * FROM weather'
+        if weatherex[1] != weather and weatherex[2] != temp:
+            amount -= 1
+            sql = f'insert into weather(status,tempature) values("{weather}","{temp}")'
+            my_cursor.execute(sql)
+            connector.mydb.commit()
+
+
+weather_adder(weather_list)
