Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>DB_HOST = localhost\r\nDB_DATABASE = flight_game\r\nDB_USER = root\r\nDB_PASSWORD = password\r\n\r\nFLIGHT_RANGE = 2778  # km
===================================================================
diff --git a/.env b/.env
--- a/.env	
+++ b/.env	
@@ -1,6 +1,4 @@
 DB_HOST = localhost
 DB_DATABASE = flight_game
 DB_USER = root
-DB_PASSWORD = password
-
-FLIGHT_RANGE = 2778  # km
\ No newline at end of file
+DB_PASSWORD = 3042
